### ISSUE TITLE
Visualizing annotations

### ISSUE THREAD

**Issue by gcelano** [webanno/webanno#491]
it would be highly beneficial if annotations could be viewed by anyone without logging in

**Comment by sholschneider** [comment]
It might be beneficial but shouldn't be the default setting so you can still prevent annotators from copying their colleagues' work.

**Comment by gcelano** [comment]
I agree. Probably just a feature which can be activated or deactivated

**Comment by reckart** [comment]
I think this could be like a "visibility" setting on the project level which could be "private" or "public".

**Comment by kazukisan** [comment]
This feature request is for having an option the project open to anybody or not.
If it is open and once the user lands the page, I believe that's when features from #467 will allow the user to specify whose annotation the person wants to look at.
